.travis.yml

```diff
@@ -1,10 +1,18 @@
 language: node_js
 services: mongodb
 sudo: required
-node_js:
-  - 10
-  - 12
-  - 14
+matrix:
```

Comment:
I've seen few times that linting can actually fail under one node.js version and pass on another.
I'd recommend removing the "avoid running it 3 times". Let's lint on each node.js.

Also, I'd recommend us to do more KISS principle. Simplify not for machines but for humans (us). In other words, this was a premature optimisation. Let's remove the `matrix` entirely.

Also, as it was reported on a screenshot, the tests do not actually run under node 14.

Reply:
Yep, fair points! Feel free to modify unless I get to it sooner. :-)

Reply:
@koresar take a look, I updated the travis config.
I tested if a lint-error would fail the build here: https://travis-ci.org/github/agenda/agenda/builds/754085260